Rubberduck.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.26823.1
```

Comment:
We haven't upgraded to C# 7 yet. CI builds runs C# 6.0.

Reply:
Darn. I was starting to enjoy VS 2017.